.werft/integration-tests-startup.yaml

```diff
@@ -0,0 +1,100 @@
+pod:
+  serviceAccount: werft
+  nodeSelector:
+    dev/workload: builds
+  imagePullSecrets:
+  - name: eu-gcr-io-pull-secret
+  volumes:
+  - name: gcp-sa
+    secret:
+      secretName: gcp-sa-gitpod-dev-deployer
+  - name: config
+    emptyDir: {}
+  containers:
+  - name: gcloud
+    image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:prs-ws-man-bridge-audit.10
+    workingDir: /workspace
+    imagePullPolicy: Always
+    env:
+    - name: NODENAME
+      valueFrom:
+        fieldRef:
+          fieldPath: spec.nodeName
+    volumeMounts:
+    - name: gcp-sa
+      mountPath: /mnt/secrets/gcp-sa
+      readOnly: true
+    - name: config
+      mountPath: /config
+      readOnly: false
+    command:
+    - bash
+    - -c
+    - |
+      set -euo pipefail
+
+      function cleanup ()
+      {
+        werft log phase "cleanup preview environment" "cleanup preview environment"
+        BUILD_ID=$(werft run github -a namespace=staging-gitpod-test-main -j .werft/wipe-devstaging.yaml github.com/gitpod-io/gitpod:main)
+
+        echo "start cleanup preview environment, job name: ${BUILD_ID}" | werft log slice "cleanup"
+        werft log result -d "cleanup job" url "https://werft.gitpod-dev.com/job/${BUILD_ID}"
+      }
+
+      echo "preparing config." | werft log slice prepare
+      gcloud auth activate-service-account --key-file /mnt/secrets/gcp-sa/service-account.json
+      export GOOGLE_APPLICATION_CREDENTIALS="/home/gitpod/.config/gcloud/legacy_credentials/cd-gitpod-deployer@gitpod-core-dev.iam.gserviceaccount.com/adc.json"
+      export DOCKER_HOST=tcp://$NODENAME:2375
+      echo "copied config..." | werft log slice prepare
+      werft log slice prepare --done
+
+      werft log phase "build preview environment" "build preview environment"
+      # this is because we don't want to increate gitpod-build-main number, we can choose a name
+      sudo cp .werft/build.yaml .werft/test.yaml
+      # version is actual namespace that werft is build, --job-file should be same with above
+      VERSION=$(werft run github -j .werft/test.yaml -s .werft/test.yaml github.com/gitpod-io/gitpod:main)
```

Comment:
if you use `--follow-with-prefix`, the `run` will wait until the other job is done and integrate the log output correctly.